### PR TITLE
[TIMOB-26047] iOS: Fix Fix launch-screen + forceSplashAsSnapshot combination, fix related unit-test

### PR DESCRIPTION
**Pull request by hansemannn**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-26047

### CODE REVIEW

**Review by vijaysingh-axway — APPROVED** [review]
CR passed.

File: tests/Resources/ti.app.test.js [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+	it.ios('disableNetworkActivityIndicator', function () {
+		should(Ti.App.disableNetworkActivityIndicator).be.a.Boolean;
+	});
+
+	it.ios('#getDisableNetworkActivityIndicator()', function () {
+		should(Ti.App.getDisableNetworkActivityIndicator).be.a.Function;
+		should(Ti.App.getDisableNetworkActivityIndicator()).be.a.Boolean;
+	});
+
+	it.ios('forceSplashAsSnapshot', function () { // eslint-disable-line mocha/no-skipped-tests
```
Comment (sgtcoolguy) [inline tests/Resources/ti.app.test.js]: remove the trailing eslint comment, since it's no longer needed

File: tests/Resources/ti.app.test.js [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+		should(Ti.App.forceSplashAsSnapshot).be.false;
+
+		Ti.App.forceSplashAsSnapshot = true;
+		should(Ti.App.forceSplashAsSnapshot).be.true;
+
+		Ti.App.forceSplashAsSnapshot = false;
+		should(Ti.App.forceSplashAsSnapshot).be.false;
+	});
+
+	it.ios('#getForceSplashAsSnapshot()', function () { // eslint-disable-line mocha/no-skipped-tests
```
Comment (sgtcoolguy) [inline tests/Resources/ti.app.test.js]: remove the trailing eslint comment, since it's no longer needed

**Review by sgtcoolguy — APPROVED** [review]
LGTM

**Pull request merged**